courtbot/states/CA/__init__.py

```diff
@@ -0,0 +1,123 @@
+
+import requests
+
+"""A package for retreiving court calendar data from Santa Clara County, California"""
+
+counties = [ 'santa clara' ]
+
+courts = [ 'santa clara' ]
+
+court_names = ["Civil",
+               "Probate",
+               "Family",
+               "Juvenile",
+               "Criminalhoj",
+               "CriminalPaloAlto",
+               "CriminalSouthCounty"]
+
+class Court:
+
+    def __init__(self, name, department):
+        self.__name = name
+        self.__dept = department
+
+    def setDepartment(self, dept_name):
+        self.__dept = dept_name
+
+    def getDepartment(self):
+        return self.__dept
+
+class CourtAppearance:
+
+    def __init__(self):
+        self.__when = ''
+
+    def __call__(self, appear_date, case_num):
+
+        cases = get_list( appear_date )
+
+        for case in cases:
+            if case['caseNbr'] == case_num:
+                return case
+
+        return {}
+
+    def setCourtName(self, name):
```

Comment:
These getter/setter + property combinations: 
```
   def setCourtName(self, name):
        self.__court_name = name

    def getCourtName(self):
        return self.__court_name

    court_name = property(getCourtName, setCourtName)
```
seems really an overkill, esp. , if you are not doing anything in the getter/setter methods. 

A cleaner way is to just add a instance variable in for the class: 
```
def __init__(self):
   self.court_name = ''
```

In the code, you can simply do:
```
court_app = CourtAppearance()
print("{}".format(court_app.court_name))
```

Reply:
"seem really an overkill" - yes, I agree. I do not "grok" the structure of the app here and am trying to make one thing to be like another thing when I do not completely understand the other thing.

"you can simply do" - yes, I suspect you are right. But if I look at that second line of code, the print statement seems odd to me. It is obviously concise. It just is not particularly meaningful to me. We will see.

Reply:
What I mean is - a usage is - an instantiation of the object, then reference to the variable that object contains, using `print` to display that variable.